tests/unit/frontend/tvm/relay_tools/test_relay_l3_math_and_transform.py

```diff
@@ -404,3 +404,31 @@ def test_zeros_like(self):
         assert layers[0].type[0] == 'Constant'
         assert layers[1].type[0] == 'AnyOp'
         assert layers[1].shapes == [1, 6, 4, 4]
+
+
+        
+    @unittest.skipIf(skip, "Could not import TVM and/or TVM frontend")
+    def test_arange_full_and_reshape(self):
+        start = relay.expr.const(0.0)
+        stop = relay.expr.const(10.0)
+        step = relay.expr.const(1.0)
+
+        fill_val = relay.expr.const(1.0)
+        fill_shape = [10, 1]
+        dtype = 'float32'
+
+        left = relay.arange(start, stop, step, dtype)
+        left = relay.reshape(left, [-1, 1])
+        left = relay.reshape(left, [1,-1])
+        
+        right = relay.full(fill_val, fill_shape, dtype)
+        right = relay.reshape(right, [1,-1])
+
+        net = relay.multiply(left, right)
+
+        mod = tvm.IRModule.from_expr(net)
+        params = {}
+        xgraph = xf_relay.from_relay(mod, params)
+        layers = xgraph.get_layers()
+
+        assert layers[0].type[0] == 'Constant'
```

Comment:
Could you add more checks? For example check whether number of layers are correct, the shapes are correct, etc

Reply:
Done